app/models/search/bookmark_search_form.rb

```diff
@@ -181,4 +169,26 @@ def sort_direction(sort_column)
     'desc'
   end
 
+  private
+
+  def processed_options(opts={})
+    [:date, :bookmarkable_date].each do |countable|
+      if opts[countable].present?
```

Comment:
Was there a reason we originally wrote this the way we did instead of chaining the `gsub` methods? I have this vague feeling there might have been, but can't quite recall.

(Doesn't need any action, Elz, just kind of making a note in case we notice something hinky after this is deployed... or in case someone has a better memory.)

Reply:
I doubt it, and if we're trying to freeze strings for future Ruby compatibility, we should avoid methods like gsub! that change them in place.